app/routes/lending/LendableObjectsList.tsx

```diff
@@ -0,0 +1,128 @@
+import { usePreparedEffect } from '@webkom/react-prepare';
+import qs from 'qs';
+import { Helmet } from 'react-helmet-async';
+import { Link, useHistory, useLocation } from 'react-router-dom';
+import Card from 'app/components/Card';
+import { Content } from 'app/components/Content';
+import TextInput from 'app/components/Form/TextInput';
+import { Image } from 'app/components/Image';
+import NavigationTab from 'app/components/NavigationTab';
+import type { ListLendableObject } from 'app/store/models/LendableObject';
+import styles from './LendableObjectsList.css';
+
+const LendableObject = ({
+  lendableObject,
+}: {
+  lendableObject: ListLendableObject;
+}) => {
+  return (
+    <Link to={`/lending/${lendableObject.id}`}>
+      <Card isHoverable hideOverflow className={styles.lendableObjectCard}>
+        <Image
+          className={styles.lendableObjectImage}
+          src={lendableObject.image}
+          alt={`${lendableObject.title}`}
+        />
+        <div className={styles.lendableObjectFooter}>
+          <p>{lendableObject.title}</p>
```

Comment:
Why is the title in a paragraph tag? I see that the styling from `.lendableObjectFooter` is ment for this element, so might as well move it to this directly. The div can be a Flex component